format.c

```diff
@@ -27,24 +36,37 @@ fd_strplcpy(char* restrict dst, const char* src, size_t size) {
 
 static char*
 fd_strpcat(char* restrict dst, struct FdStr src) {
+#ifdef _MSC_VER
+    unsigned lim = src.sz;
+    for (unsigned i = 0; i < lim; i++)
+        *dst++ = src.s[i];
+    return dst;
+#else
     unsigned lim = __builtin_constant_p(src.sz) && src.sz <= 8 ? 8 : 16;
     for (unsigned i = 0; i < lim; i++)
         dst[i] = src.s[i];
     // __builtin_memcpy(dst, src.s, 16);
     return dst + src.sz;
+#endif
 }
 
 static unsigned
 fd_clz64(uint64_t v) {
+#ifdef _MSC_VER
+    unsigned long index;
+    _BitScanReverse(&index, v);
+    return 31 - index;
```

Comment:
This looks wrong... the input is a 64-bit integer, so clearly the MSB can be in the upper 32 bits.

Reply:
Oops, I somehow thought this was a 32-bit quantity while porting this part. Not sure how I got that mixed up.